[Release] cxxrandom v2.0.1

cppcooper opened the pull request:
There's been a print line enabled that should have been disabled long ago. Basically just getting rid of a message that floods the console.

lethosor commented:
Trying to re-run Travis

lethosor commented:
Could you change the line endings back to Unix-style (\n)? It's difficult for me to do in the web interface, but the current \r\n endings you're using aren't allowed by our style guide, and make it hard to tell what actually changed here (since the version in this repo uses \n).

cppcooper commented:
oooh, I couldn't figure out why it was showing the entire file changed.

cppcooper commented:
May need to drop the merge message since I added it manually after unwinding the commits.

lethosor commented:
I don't know what you did, but now there are a ton of other changed files. Do you have another branch that isn't cherry-picked/rebased/merged/modified like this one?

cppcooper commented:
You wanted to be able to see the changes, so I corrected the line endings on my end. In order for you to see the changes though it meant I needed to `git reset --soft HEAD^` a couple times. Which meant I had the merge commit popped off the list, and had to manually make it. Anyways I've removed it, I'm pretty sure I've still got the merge in there, just now there won't be a message indicating it.

lethosor commented:
Yeah, I think whatever merge you had made wasn't quite right, which is why there were a lot of changes listed (including some conflicts). It looks like just a single line was commented out now. Is that right? I'm definitely fine with merging this if so.

cppcooper commented:
Taking a look just now verifying where the branch is at. Currently master isn't merged, all the commits that were merged in were undone with the merge commit. Looks like that single commit stitches all the other commits to it. Anyways, so what happened before was basically equivalent to a `git merge --squash` with a message. It had all the same changes but since it had none of the original commits.

Dunno if that clears things up for you, but things should be fine now.
